compiler/ballerina-parser/src/test/resources/types/func-type/func_type_source_13.bal

```diff
@@ -0,0 +1,2 @@
+function foo ) {
+}
```

Comment:
Add a new line. Please fix in other .bal files as well

Reply:
Fixed.